src/main/java/org/folio/services/VoucherLinesRetrieveService.java

```diff
@@ -20,7 +20,7 @@
 import io.vertx.core.Context;
 
 public class VoucherLinesRetrieveService {
-  static final int MAX_IDS_FOR_GET_RQ = 15;
+  static final int MAX_VOUCHER_IDS_FOR_GET_RQ = 10;
```

Comment:
I think there is no need to get away from project's common chunk size. At least I don't see any critical reasons to do that